Dockerfile

```diff
@@ -46,36 +41,17 @@ RUN apk add --update --no-cache git g++ linux-headers
 
 ARG USER
 ARG config_file=environments/docker.toml
-ARG vdb_connect
-
-# setup environment
-ENV GOPATH $HOME/go
-ENV GO111MODULE on
```

Comment:
how come we're able to remove these ENV variables? Don't we still need to make sure that the GOPATH is set properly, and go mods are on?

Reply:
Don't need them since we built the plugin in the builder container, so this container doesn't need to peruse the GOPATH or load modules